docs/articles/2025/20250117-spatial.md

```diff
@@ -7,7 +7,7 @@ The Census team is pleased to announce the release of Spatial data on Census!
 
 This has been a large joint effort between the Census team and TileDB to create an easy-to-use and backwards-compatible spatial schema that enables easy analysis with non-spatial and spatial data!
 
-This first release is a **beta release** with the ability to export all spatial data (10x Visium and Slideseq) from an `obs/var` query in `SpatialData`. The ability to export `SpatialData` with spatial-based filters, transforms, etc., is not supported and will be implemented in a future release.
+This first release is a **beta release** with the ability to export all spatial data (10x Visium and Slideseq) from an `obs/var` query to {class}`~spatialdata.SpatialData` ([spatialdata.scverse.org](https://spatialdata.scverse.org/en/stable/)). The ability to export `SpatialData` with spatial-based filters, transforms, etc., is not supported and will be implemented in a future release.
```

Comment:
Should we specify V2?

Reply:
in reference to the mention of SlideSeq

Reply:
Ah, yes!

```suggestion
This first release is a **beta release** with the ability to export all spatial data (10x Visium Spatial Gene Expression and Slide-seqV2) from an `obs/var` query to {class}`~spatialdata.SpatialData` ([spatialdata.scverse.org](https://spatialdata.scverse.org/en/stable/)). The ability to query census with spatial-based filters, transforms, etc., is not supported and will be implemented in a future release.
```

I went with the more official names for both assays, and also made a little change here on the export language (found it a little confusing when reading it).